data/alice/config.toml

```diff
@@ -0,0 +1,185 @@
+[API]
```

Comment:
Just noting it here for the future. Before merging and getting this ready for production, we should probably move these configs to a `scripts/taproot` that we can use for testing purposes, or remove them. Also, we can probably remove all the commented lines if they are already in the config template.

Reply:
We agreed to move everything that is under `data` will be move in Minio and downloaded before starting the demo. Those are pre-generated keys and config only required when developing/running the demo.